shared/src/test/diff/ecoop23/PolymorphicVariants.mls

```diff
@@ -198,16 +198,16 @@ Test2.eval(Cons(("a", Abs("d", Var("d"))), Nil), Add(Numb(1), Var("a")))
 
 module Test3 extends EvalVar, EvalExpr, EvalLambda
 //│ module Test3 {
-//│   fun eval: forall 'a 'A. (Cons[{_1: anything, _2: 'result}] | Nil, Abs['b] | App['A] | Object & 'c & ~#Abs & ~#App) -> ('A0 | 'a)
+//│   fun eval: forall 'a. (Cons[{_1: anything, _2: 'b}] | Nil, Abs['c] | App['d] | Object & 'e & ~#Abs & ~#App) -> ('A | 'a)
```

Comment:
I need to investigate this change. Why would your modifications make one less variable be generalized in this signature? (It means it'll be generalized at the module level instead!)

Reply:
Ok, I guess what happens here is some obscure interaction due to the fact that the new typing of patterns now involves intermediate type variables. I think it's likely a minor thing and this can be closed.